app/_src/kubernetes-ingress-controller/reference/version-compatibility.md

```diff
@@ -78,7 +78,7 @@ For each {{site.kic_product_name}} release, tests are run to verify this documen
 | Istio 1.21                | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> |
 | Istio 1.22                | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> |
 | Istio 1.23                | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> |
-| Istio 1.24                | <i class="fa fa-times"></i> | <i class="fa fa-times"></i> | <i class="fa fa-times"></i> | <i class="fa fa-times"></i> | <i class="fa fa-check"></i> |
```

Comment:
Looks like the Istio table is still missing 3.4.x @mlavacca

Reply:
Good catch. Should be good now 👍🏻